fmradioiriscontrol.cpp

```diff
@@ -490,10 +516,29 @@ void FMRadioIrisControl::cancelSearch()
 
 void FMRadioIrisControl::start()
 {
+    initRadio();
 }
 
 void FMRadioIrisControl::stop()
 {
+    if (m_currentFreq > 0)
+        m_pendingFreq = m_currentFreq;
+
+    m_timer->stop();
+    if (m_workerThread) {
+        m_workerThread->setQuit();
+        SetCtrl(V4L2_CID_PRIVATE_IRIS_STATE, 0); // assuming this wakes blocking ioctl()
```

Comment:
So guess this will trigger tuner signaled as not available. Suppose it's fine, but pondering if the member variable should be set just in case.